sdk/servicebus/service-bus/test/utils/envVarUtils.ts

```diff
@@ -15,18 +15,16 @@ export enum EnvVarKeys {
 }
 
 /**
- * Utility to retrieve the environment variable value based
- * on targetted platform and type.
- *
+ * Utility to retrieve the environment variable value with given key.
```

Comment:
Excuse my overpedantry, but environment variables have names not keys 🤷‍♂️

Reply:
Updated!